powerline/segments/i3wm.py

```diff
@@ -45,6 +47,8 @@ def workspaces(pl, only_show=None, output=None, strip=0):
 		else:
 			conn = i3ipc.Connection()
 
+	output = output or ('output' in segment_info and segment_info['output'])
```

Comment:
Do not use weird construct `key in dict and dict[key]`, there is `dict.get(key, False)` which replicates the behaviour exactly (though you probably meant `dict.get(key)` which is the same as `dict.get(key, None)`).